src/components/button/button.scss

```diff
@@ -19,9 +19,6 @@
 }
```

Comment:
Colours in these examples in the Action button guidelines are broken:

This PR:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/154725/211844576-bf4e0e9e-f902-449b-bf8a-c3c4f93d1e63.png">
<img width="400" alt="image" src="https://user-images.githubusercontent.com/154725/211845021-5d8ea884-4be4-461d-b799-ab8f2343c756.png">


Latest `next`:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/154725/211844860-c5bb5cf1-60b7-480a-b1ae-bb0c37da7d78.png">
<img width="400" alt="image" src="https://user-images.githubusercontent.com/154725/211845081-39e4e3c8-45fc-4a65-8e30-4c67cbb33111.png">

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/211847329-ea2991e2-3418-4d4a-90f7-22437384b3a5.png">

Reply:
Ah good catch. Thanks! I'll fixup soon

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/211854931-7be1708a-6d99-4772-86e3-725ce7b8d231.png">

Reply:
(Note the text colour of the last button in this last screenshot.)

Reply:
This seems to be due to this commit 94663859f38863d4a88850154d8035b7b3cec74e

I'm not sure if I understand why we should import the lime-theme into each and every component. Currently too tired to figure out why. 

But adding the following to `button.scss` will fix this issue:
```scss
@use '../_theme-color-variables';
:host {
    @include theme-color-variables.theme-color-variables;
}
```

But I don't know if other components are affected by the same thing. So maybe we should simply droip this commit. 
I also realizes that I effed up a little bit in this commit and have some of the removal of old lime brand colors which should be on the next commit (a73dbb74ef222c6a1d99bbdcc0a6e524c47ff8d6) in here by mistake.

Reply:
> I'm not sure if I understand why we should import the lime-theme into each and every component. Currently too tired to figure out why.

Yeah, that's a bit wierd…

Maybe we aren't actually setting the variables on `:root` in the docs? 🤔

> But adding the following to button.scss will fix this issue:

That seems like a good solution, rather than doing it everywhere, like we did before.

> But I don't know if other components are affected by the same thing. So maybe we should simply droip this commit.

I'm looking through literally _every_ example in the docs, comparing side-by-side with the latest `next`, so if anything else is affected, I'll hopefully catch it 😊

Reply:
> I also realizes that I effed up a little bit in this commit and have some of the removal of old lime brand colors which should be on the next commit (https://github.com/Lundalogik/lime-elements/commit/a73dbb74ef222c6a1d99bbdcc0a6e524c47ff8d6) in here by mistake.

If you haven't already fixed that, I can do it for you, so we don't get competing force-pushes again 😄

Reply:
⚡